ext/syncthing/src/github.com/syncthing/syncthing

```diff
@@ -1 +1 @@
-Subproject commit 668eb7c398c2e8d0b9f7beb319b7789681d7d89f
+Subproject commit 6d280e7b6461d407c90f8706e348393c657e995f
```

Comment:
Regarding the static finals: I am not 100% shure if the corresponding class is loaded if you use a static final field of that class. Technically it would not be necessary (if the initial value is a constant), but I did not find a clear reference for that in the web. So I thought defensive and would put the variable to the object that is living the longest fime ;-)

Regarding the dummy method: My thought was, that the later implementation of that dummy method would not change the other code. So a later change would be more focused.
But looking at the whole change it would make no difference - you have to make a change at both places.

So I forget about the dummy method for now.